ConsoleApp6/Packaging.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace KATA
+{
+    class Packaging : IDecorater
+    {
+        IPaymentsServices payments;
```

Comment:
as we understand from task number 6, the costs are with open types, meaning we may have to pay for different types of costs in the feautre.
I don't see we have to make a standalone class for each cost, as all costs are with desc, amount, and type, with the same calculation criteria, so why have them separated? no need to be specialized here